mass_mailing_resend/views/mass_mailing_views.xml

```diff
@@ -12,6 +12,16 @@
                         type="object"
                 />
             </field>
+            <header position="after">
+                <div class="alert alert-warning oe_text_center" attrs="{'invisible': ['|', ('state', '!=', 'draft'), ('sent', '=', '0')]}">
```

Comment:
Following the flow: shouldn't this be more like:
```python
{'invisible': ['|', ('state', '=', 'done'), ('state', '!=', 'done'), ('sent', '=', '0')]}
```
So it's only visible when a mailing is reset to draft and it has sent mails. Otherwise it's visible at the first composition.

Reply:
No, it's correct as it's now: it's only visible when on draft and sent > 0.

Reply:
Isn't it when draft **or** sent = 0

Reply:
I don't get you. Please re-read my comment

Reply:
Ok, yeah. I got it wrong because the second condition doesn't seem to be working fine. When you create a new draft mailing the bubble is visible (maybe is False until the field is computed by the first time?)

Reply:
Ok, it was more subtle than that :smile: The quotes in `('sent', '=', '0')` are avoiding proper evaluation. So, this should work:
```python
{'invisible': ['|', ('state', '!=', 'draft'), ('sent', '=', 0)]}
```

Reply:
Agh...